### PR TITLE
add optional auto height + fix list widget size

### PR DESCRIPTION
**Pull request by Bars92**
added auto hight flag to force auto height when using a layout report or not added missing padding to section
fixed list widget does not stretch full width

### CODE REVIEW

File: reportService/reportsServer.js [hunk 1]
```diff
@@ -66,7 +66,8 @@ const PAGE_MARGIN = 60;
   const headerRightImage = process.argv[9] || '';
   const pageSize = process.argv[11] || PAGE_SIZES.Letter;
   const disableHeaders = process.argv[12] === true || process.argv[12] === "true";
-  const chromeExecution = process.argv[13] || paths['chromium'] || paths['google-chrome-stable'] || paths['google-chrome'] || '/usr/bin/chromium-browser';
+  const forceAutoHeightLayout = process.argv[13] === true || process.argv[13] === "true";
```
Comment (dorsha) [inline reportService/reportsServer.js]: Why did you change the order? Why not adding the new one as 14? This will break compatibility.

File: src/components/Layouts/ReportLayout.js [hunk 2]
```diff
@@ -75,14 +77,20 @@ class ReportLayout extends Component {
             if (maxHeight < item.element.clientHeight) {
               maxHeight = item.element.clientHeight;
             }
-            item.element.style.top = `${accumulatedHeight}px`;
+            if (isAutoHeightLayout) {
```
Comment (dorsha) [inline src/components/Layouts/ReportLayout.js]: Default is false now right? This will change existing behavior  when not passing auto height true.

File: reportService/reportsServer.js (hunk 1 above)
Comment (Bars92) [reply reportService/reportsServer.js]: i think we can assume `chromeExecution` is not used yet (only by the server) and the `forceAutoHeightLayout` should be before. i can switch it if we fear bc

File: src/components/Layouts/ReportLayout.js (hunk 2 above)
Comment (Bars92) [reply src/components/Layouts/ReportLayout.js]: yeah default is false, while investigation reports pass true.

Comment (dorsha) [reply src/components/Layouts/ReportLayout.js]: I'm talking about other users who use this.. the behavior will be changed for them.

File: reportService/reportsServer.js (hunk 1 above)
Comment (dorsha) [reply reportService/reportsServer.js]: Let's switch.
